### ISSUE TITLE
Steady Flight Only showing up on the wrong mounts

### ISSUE THREAD

**Issue by frantisekmachek** [exochron/MountJournalEnhanced#122]
In 11.0 they added "Steady Flight Only" labels to the mount tab but with this addon, the labels show up under the wrong mounts (sometimes even ground mounts).

**Comment by exochron** [comment]
Thanks for reporting this. I was somehow missing that label. So in theory it should work again (with version 2.39.0 onward).

However, it seems that the mount api from the game is also broken. On my client it only labels the [Otterworldly Ottuk Carrier] as steady flight.

Would you please update and check, if you got the Ottuk labeled as well? :)

**Comment by frantisekmachek** [comment]
Yes, the Ottuk is labeled correctly now, thank you for the quick fix.

I might be wrong, but it seems like the Ottuk is actually the only flying mount incapable of skyriding and it's not just a bug. I don't know why that's a thing when even mounts like the Wildseed Cradle can skyride. I checked the forums as well and mounts that were reported to be unable to skyride have been updated since, leaving the Ottuk to be the only one left.
